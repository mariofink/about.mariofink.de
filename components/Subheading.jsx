const SubHeading = ({ children }) => (
  <h2 className="mt-6 mb-3 text-3xl font-bold">{children}</h2>
);

export default SubHeading;
