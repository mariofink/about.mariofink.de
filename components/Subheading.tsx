interface props {
  children: React.ReactNode;
}

const SubHeading = ({ children }: props) => (
  <h2 className="mt-6 mb-3 text-3xl font-bold">{children}</h2>
);

export default SubHeading;
