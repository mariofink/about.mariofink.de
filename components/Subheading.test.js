import React from "react";
import { render } from "@testing-library/react";
import Subheading from "./Subheading";

describe("Subheading", () => {
  it("renders", () => {
    const { container } = render(<Subheading>My Subheading</Subheading>);
    expect(container).toMatchSnapshot();
  });
});
