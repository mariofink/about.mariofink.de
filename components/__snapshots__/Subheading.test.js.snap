// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subheading renders 1`] = `
<div>
  <h2
    class="mt-6 mb-3 text-3xl font-bold"
  >
    My Subheading
  </h2>
</div>
`;
